### PR TITLE
add Chinese translation

### PR DESCRIPTION
**Pull request by haohanyang**
# GitHub Resume Generator

GitHub Flow PR Image

Thank you so much for opening a pull request! :smiley:

Please provide the following information where possible to help the pull request reviewer merge your PR quicker!

## Relevant Project Issue Numbers :hash:

_Add any related issue numbers from the GitHub project. Append the `#` symbol before issue numbers._

> e.g. #58

#15 
## Involved Project Members :bust_in_silhouette:

_Tag any GitHub usernames involved or directly responsible for reviewing the proposed changes. Append the `@` symbol before usernames._

> e.g @Charlotteis

@sabesansathananthan 
## An Explanation of Your Changes :speech_balloon:

Add Chinese translation

## For transilators.

- [x] Create a file called `README.lang.md` in the translations folder.
  lang means shorten form of your language Refer this [site](https://meta.wikimedia.org/wiki/Template:List_of_language_names_ordered_by_code).

- [x] Check all the links are working.

- [x] Check all the parts are translated.
- [x] If about and built-with are not working then inspect the page and copy the href and paste in your file.
- [x] Edit the translations/Translations.md adding you language icon/link
- [x] Edit the main README.md adding your language icon

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,8 @@
 <kbd>[<img title="Indonesian" alt="Indonesian" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/id.svg" height="14">](./translations/README.id.md)</kbd>
 <kbd>[<img title="Sinhala" alt="Sinhala" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/lk.svg" height="14">](./translations/README.si.md)</kbd>
 <kbd>[<img title="Bulgarian" alt="Bulgarian" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/bg.svg" height="14">](./translations/README.bg.md)</kbd>
+<kbd>[<img title="Chinese" alt="Chinese" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/cn.svg" height="14">](./translations/README.zh.md)</kbd>
```
Comment (sabesansathananthan) [inline README.md:32]: add the specific Chinese extension. please search your language and add the correct code...
zh or zh-classical or cdo or gan or hak 

 https://meta.wikimedia.org/wiki/Template:List_of_language_names_ordered_by_code

**Review by sabesansathananthan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sabesansathananthan) [reply README.md:32]: fixed

File: translations/README.zh.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+4. 访问 <http://localhost:3000>
+
+## [在线演示](https://react-github-resume.vercel.app/)
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/sabesansathananthan/React-GitHub-Resume)
+
+## 📄 许可
+
+本项目遵照MIT许可证 - 详情请访问[LICENSE](./LICENSE)
```
[the remark this answers is not in the document]
Comment (sabesansathananthan) [reply translations/README.zh.md]: fixed

**Comment by sabesansathananthan** [comment]
@haohanyang Please fixed the request changes
- [ ] src image is viewable.
- [ ] table of contents are working
- [ ] word LICENSE translated in Chinese.
- [ ] fixed Other languages Link.

**Comment by haohanyang** [comment]
Bugs listed are fixed.

**Comment by sabesansathananthan** [comment]
Nice. I got a great Idea from your PR... Thank you... <3

**Review by sabesansathananthan — APPROVED** [review]

**Pull request merged**